Contributing.md

```diff
@@ -0,0 +1,20 @@
+# Wie kann ich mithelfen?
+Cool das Du hier bist 🎉. Hier einige Hinweise:
+
+## Ich möchte eine weitere API des Bundes dokumentieren
+Bitte lege in diesem Repository ein Issue an, damit wir das diskutieren können. Dann werden wir Dir ein Repo in der Organisation und entsprechende Domains aufsetzen. Du kannst uns auch anonym (z.B. via kontakt@bund.dev) Dokumentation zukommen lassen, die wir dann veröffentlichen.
```

Comment:
Finde diese Version super und sie beantwortet alle Fragen welche ich hätte 👍 
Anbei nur eine Mini-Umformulierungsempfehlung, welche es vielleicht **noch** expliziter macht, wo und wie man beginnen kann.

```suggestion
Bitte lege im Sofortmassnahmen Repository ein Issue an, damit wir das diskutieren können. Dann werden wir Dir ein Repo in der Organisation und entsprechende Domains aufsetzen. Du kannst uns auch anonym (z.B. via kontakt@bund.dev) Dokumentation zukommen lassen, die wir dann veröffentlichen.
```

Reply:
Vielleicht noch eine weitere Idee aber nichts dringendes: Das `Sofortmassnahmen`-Repo könnte als erstes Repo in der Organization [gepinned](https://stackoverflow.com/a/67478102) werden. Dieses Repo ist ja der Startpunkt, um die anderen Projekte und die Beteiligungsmöglichkeiten zu verstehen. Daher wäre ein prominenter Platz vielleicht hilfreich.